webapp/app/templates/macros.html

```diff
@@ -85,7 +84,7 @@ <h1 class="my-4">{{ title }}</h1>
             {% elif field.type == 'RadioField' %}
                 {{ components.radio_field(field) }}
             {% else %}
-                {% if field.errors %}
+                {% if first_field or field.errors %}
                     {{ field(class=classes, autofocus=True) }}
                 {% else %}
                     {{ field(class=classes) }}
```

Comment:
Just realized: we can also write it like this:
```suggestion
                {{ field(class=classes, autofocus=first_field or field.errors) }}
```
Not sure if that's cool though. This way we'd have an autofocus attribute on each field - suppose that's not how it's supposed to be?

Reply:
I think this is a great idea. As autofocus is an attribute which is either present or not, I would assume that is is not set for False. I'll look into it but this is a very clean solution.

Reply:
Unfortunately, it did not work for me. It did not seem to take the boolean value

Reply:
That's odd. I tried it because I was curious and the attribute was correctly set? 🤔

Reply:
That's really strange. I'll have another look.

Reply:
Ok, I think that the reason is that Safari and Chrome handle the HTML different. If we use the inline boolean, this will result in the following HTML:

`<input autofocus ...>` if the boolean is `True`
`<input autofocus=() ...>` if the boolean is not `True`

Chrome will still focus the first field, while Safari will focus the last field. I guess because Safari only checks whether autofocus is present and ignores that a value has been set. Unfortunately, it also does not matter if we set it to False or True, the value will be set anyway.

Reply:
Okay, nice, so great that you investigated that, thank you! :)